morphos/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: morphos_web_1
+      APP_PORT: 8080
+
+  web:
+    image: ghcr.io/danvergara/morphos-server:0.2.0
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 8080:8080
```

Comment:
We don't need to expose a port on the host here. Instead, the app_proxy container + docker's internal dns handles this. You can delete the entire ports directive.

Reply:
Gotcha!
It's gone!